src/wm/background.vala

```diff
@@ -140,6 +140,71 @@ public class BudgieBackground : Meta.BackgroundGroup
         }
     }
 
+    /**
+     * call accountsservice dbus with the background file name
+     * to update the greeter background if the display
+     * manager supports the dbus call.
+     */
+    void set_accountsservice_user_bg(string background)
+    {
+        DBusConnection bus;
+        Variant variant;
+
+        try
+        {
```

Comment:
Trying to enforce a "new" (Josh) style everywhere, let's have curly braces on same line, single space before it :smile: